src/drawing/DrawingCreator.tsx

```diff
@@ -145,6 +146,20 @@ export default function DrawingCreator({
         };
     }, [drawingStatus]); // eslint-disable-line react-hooks/exhaustive-deps
 
+    React.useEffect(() => {
+        const { current: element } = creatorElRef;
+
+        if (!element) {
+            return;
+        }
+
+        const svg = encodeURIComponent(DrawingCursorTemplate.replace('{{cursorFillColor}}', color).replace(/"/g, "'"));
+
+        Object.assign(element.style, {
+            cursor: `url("data:image/svg+xml,${svg}") 6 22, crosshair`,
```

Comment:
Where do the values `6` and `22` come from? Can those values be encapsulated in the DrawingCursor module?